src/Components/DrillListPage.test.js

```diff
@@ -39,7 +39,7 @@ describe('<DrillListPage />', () => {
         image: 'http://www.liberte-fitness.fr/sites/default/files/styles/slider/public/news/tone.jpg?itok=505bme2a',
         description:
           'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec scelerisque dignissim felis, at tristique mi efficitur a. Nulla pellentesque odio nunc, ut sodales ex pulvinar at. In hac habitasse platea dictumst. Pellentesque sodales nisl lorem, ac lacinia nisl lacinia eu. Proin lacinia viverra mauris, et pharetra ipsum tempus eget. Ut maximus sapien in hendrerit ultricies. Maecenas vel diam tincidunt, consequat nunc a, mattis eros. Curabitur a eros a nulla malesuada gravida. Donec massa quam, suscipit eget efficitur et, euismod sed nulla. Morbi mattis et magna a aliquam. Fusce pellentesque vel erat eget',
-        nbPlayers: 2,
+        minimalPlayersNumber: 2,
```

Comment:
`count` is more precise since it refers to something countable and not just a random number.

I think `number` makes more sense for a french developer since it's closer to the french word `nombre`

I'm ok with count